services/smtp-bridge-emulator/apihandlers/send-email.go

```diff
@@ -0,0 +1,187 @@
+package apihandlers
+
+import (
+	"errors"
+	"log/slog"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	mw "github.com/case-framework/case-backend/pkg/apihelpers/middlewares"
+
+	messagingTypes "github.com/case-framework/case-backend/pkg/messaging/types"
+
+	"github.com/gin-gonic/gin"
+)
+
+const (
+	EML_FILE_EXTENSION = ".eml"
+	EMAILS_DIR         = "emails"
```

Comment:
Please make this "path" configurable via the config.yaml, so it can be simply customised.
If preferred, we can keep this as a default value, and add the option to override it through a config variable to be able to map it to a custom path on a persistent storage.

Reply:
Extend the config struct in `services/smtp-bridge-emulator/init.go`  to read the email dir from the config file, and use that to set the location. I think either we can also define a default location (as you did, but maybe with a different name (`DEFAULT_EMAIL_DIR`) and use this if the config value is empty - or alternatively throw an error if the value is empty, to force the "user" to set a path consciously.